src/jni/types.h

```diff
@@ -51,8 +51,13 @@ namespace jni {
         template <class T>
         inline T new_global_ref(Env& env);
         void delete_global_ref(Env& p_env);
+		template <class T>
```

Comment:
formatting is weird

Reply:
Clion code style config being weird here